ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  BaseStoreKitIntegrationTests.swift
+//  PurchasesHybridCommonIntegrationTests
+//
+//  Created by Nacho Soto on 6/27/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import PurchasesHybridCommon
+
+import Nimble
+@testable import RevenueCat
+import SnapshotTesting
+import StoreKit
+import StoreKitTest
+import XCTest
+
+class StoreKit2IntegrationTests: BaseIntegrationTests {
+
+    override class var storeKit2Setting: StoreKit2Setting { return .enabledForCompatibleDevices }
+
+}
+
+class StoreKit1IntegrationTests: BaseIntegrationTests {
+
+    private var testSession: SKTestSession!
+
+    override func setUp() async throws {
+        try await super.setUp()
+
+        self.testSession = try SKTestSession(configurationFileNamed: Constants.storeKitConfigFileName)
+        self.testSession.resetToDefaultState()
+        self.testSession.disableDialogs = true
+        self.testSession.clearTransactions()
+        if #available(iOS 15.2, *) {
+            self.testSession.timeRate = .monthlyRenewalEveryThirtySeconds
+        } else {
+            self.testSession.timeRate = .oneSecondIsOneDay
+        }
+
+        // SDK initialization begins with an initial request to offerings
+        // Which results in a get-create of the initial anonymous user.
+        // To avoid race conditions with when this request finishes and make all tests deterministic
+        // this waits for that request to finish.
+        _ = try await CommonFunctionality.offerings()
+    }
+
+    override class var storeKit2Setting: StoreKit2Setting {
+        return .disabled
+    }
+
+    func testCanGetOfferings() async throws {
+        let offerings = try await CommonFunctionality.offerings()
+
+        assertSnapshot(matching: offerings, as: .json)
```

Comment:
Since the API is all about dictionaries, I believe this is the best way to test each method!